bitnami/ejbca/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ejbca
-version: 0.2.1
+version: 0.2.2
```

As it is a new feature adds more than bug fixes I would bump the chart version in a minor:
```suggestion
version: 0.3.0
```